recipes/ambertools/build.sh

```diff
@@ -0,0 +1,5 @@
+export AMBERHOME=${PWD}/amber18
+cd $AMBERHOME
+./configure  --with-python /usr/bin/python3.6 --python-install local gnu
```

Comment:
You are not using cmake here?

I'm not sure `--with-python` is really needed but in case it probably should `--with-python ${PREFIX}/python`